### ISSUE TITLE
dis2 duplicate annotation

### ISSUE THREAD

**Issue by ValWood** [pombase/pombase-chado#830]
This is driving me mad, I just can't figure out where the duplicate annotation for dis2 is coming from?
<img width="1011" alt="website ticket, duplicate" src="https://user-images.githubusercontent.com/7359272/118120969-a9f32180-b3e8-11eb-8ae2-82c52463f851.png">

The den Elzen paper has 2 chk1 substrates. This paper hasn't been through Canto so the only single annotation is in Artemis.

Any ideas?

**Comment by kimrutherford** [comment]
It looks like the same annotation is on both transcripts:
```
FT                   /GO="aspect=F; term=protein serine/threonine phosphatase
FT                   activity; GOid=GO:0004722; evidence=IDA;
FT                   db_xref=PMID:14765108;
FT                   annotation_extension=has_substrate(PomBase:SPCC1259.13);
FT                   date=20040921"
```

The website isn't smart enough to show only one.

**Comment by ValWood** [comment]
I forgot about that, also. didn't see dis2 had 2 transcripts.

So this only happens for annotations in Artemis?

**Comment by kimrutherford** [comment]
> So this only happens for annotations in Artemis?

Yep.  At the moment the contig files are the only place where you can put annotation on any transcript.

The situation with Canto loading isn't ideal at the moment as all the annotation is attached to the first (".1") transcript.

> The website isn't smart enough to show only one.

Maybe it should be showing both or at least be clear that both transcripts have the same annotation?

**Comment by ValWood** [comment]
You told me this before! OK, I will remember this time ;)

**Comment by ValWood** [comment]
Let's not worry for the moment. I'll put at low priority.  We can prioritise multi transcripts for Canto anyway. This one is already on my target list for pathways.
